### DISCUSSION TITLE
Add bin/docker-compose for Compose V2 Compatibility

### DISCUSSION THREAD

**Discussion by rangerz** [markshust/docker-magento#749] [Ideas]
@markshust 

Because [Compose V2](https://docs.docker.com/compose/#compose-v2-and-the-new-docker-compose-command) the command ... `docker-compose` to `docker compose`

Most development environments may not be ready for this change.

I think `bin/docker-compose` can help this situation and organize some util functions into this file.

But it breaks some elegantly simple logic. It's trade-off. 

For an example:
`bin/docker-compose`
```bash
#!/bin/bash

if hash docker-compose 2>/dev/null; then
  DOCKER_COMPOSE="docker-compose"
else
  DOCKER_COMPOSE="docker compose"
fi
${DOCKER_COMPOSE} -f docker-composer.yml -f docker-composer.dev.yml "$@"
```

`bin/cli`
```bash
#!/bin/bash

[ -z "$1" ] && echo "Please specify a CLI command (ex. ls)" && exit
# Orig: docker compose exec phpfpm "$@"
bin/docker-compose exec phpfpm "$@"
```

`bin/stop`
```bash
#!/bin/bash

[ -z "$1" ] && echo "Please specify a CLI command (ex. ls)" && exit
# Orig: docker compose -f docker-compose.yml -f docker-compose.dev.yml stop "$@"
bin/docker-compose stop "$@"
```

**Comment by rangerz** [comment]
Add Pull Request https://github.com/markshust/docker-magento/pull/787

**Comment by markshust** [comment]
For anyone that comes across this, this has been all merged!
